documentation/plain_rules.md

```diff
@@ -36,16 +36,15 @@ EPL is documented in [Esper website](http://www.espertech.com/esper/esper-docume
 A EPL statement to use with perseo could be:
 
 ```
-select *, "blood_rule_update" as ruleName,
-         ev.BloodPressure? as Pressure, ev.id? as Meter
+select *, ev.BloodPressure? as Pressure, ev.id? as Meter
 from pattern
- [every ev=iotEvent(cast(cast(BloodPressure?,String),float)>1.5 and type="BloodMeter")]
+    [every ev=iotEvent(cast(cast(BloodPressure?,String),float)>1.5 and type="BloodMeter")]
 ```
 
-
-* The rule name must be present with **ruleName** alias. It must be equal to the ‘name’ field of the rule object
 * The *from* pattern must name the event as **ev** and the event stream from which take events must be **iotEvent**
 * A *type=* condition must be concatenated for avoiding mixing different kinds of entities
+* The name of the rule is not mandatory, but for backward compatibility, it can be present as ** ruleName ** alias (`e.g: select *, "blood_rule_update" as ruleName...`) in the select clause. If present, it must be equal to the ‘name’ field of the rule object. If it is not present, you can still use the variable 'ruleName' in the actions, because it will be included automatically.
```

Comment:
I'd suggest split this and include part as in a a remarked "backward compatibility note". In particular I mean:

To leave this in the bullet list:

```
* The variable 'ruleName' in automatically added to the action, even if it is not present in the EPL text.
```

```
**Backward compatibility note:** since version 1.8.0 it is not mandatory to specify the name of the rule as part of the EPL text. In fact, it is not recommendable to do that. However, for backward compatibility, it can be present as *ruleName* alias (`e.g: select *, "blood_rule_update" as ruleName...`) in the select clause. If present, it must be equal to the ‘name’ field of the rule object.
```

Reply:
Fixed in 55110fb